src/js/components/AccordionPanel/AccordionPanel.js

```diff
@@ -40,35 +41,59 @@ const AccordionPanel = forwardRef(
       [active, theme.accordion.icons],
     );
 
+    // backward compatibility to the deprecated theme.accordion.hover.color
+    const headingColor =
+      theme.accordion.hover &&
+      theme.accordion.hover.color !== { dark: 'light-4', light: 'dark-3' }
```

Comment:
I don't think this comparison does what you think it does. It will always be false.

I think we just want to test for existence of `theme.accordion.hover.color`. If that exists, we use that.

Reply:
At the moment the base.js deafult is `theme.accordion.hover.color: { dark: 'light-4', light: 'dark-3' }`.
Let's say users decide to override this behavior on their theme by setting  `theme.accordion.hover.color='undefined'`,  in that case, your suggestion wouldn't be backward compatible as their setting will be overridden by `theme.accordion.hover.heading.color` which is currently `{ dark: 'light-4', light: 'dark-3' }`.

So the intended logic is if the user has defined `theme.accordion.hover.color` that is different than the default -> pick that. Otherwise, pick `theme.accordion.hover.heading.color`.

Reply:
Fixed.